gulpfile.js

```diff
@@ -311,6 +313,7 @@ function bundleTask(opts) {
       .pipe(source(opts.filename))
       // inject variables into bundle
       .pipe(replace('\'GULP_METAMASK_DEBUG\'', debug))
+      .pipe(replace('\'MASCARA_PROXY_ORIGIN\'', MASCARA_PROXY_ORIGIN))
```

Comment:
should be able to inject via envify or something (this feels a bit hacky, tho i know theres precedent the line above)